rest/api.go

```diff
@@ -356,14 +370,34 @@ func (h *handler) handlePprofThreadcreate() error {
 }
 
 func (h *handler) handlePprofMutex() error {
+	sec, err := strconv.ParseInt(h.rq.FormValue("seconds"), 10, 64)
+	if sec <= 0 || err != nil {
+		sec = 30
+	}
+	if h.server.mutexRunning.IsTrue() {
+		return base.HTTPErrorf(http.StatusForbidden, "Can only run one mutex profile at a time")
+	}
+	h.server.mutexRunning.Set(true)
+	runtime.SetMutexProfileFraction(1)
+	defer runtime.SetMutexProfileFraction(0)
```

Comment:
This should be reset prior to 'unlocking' at line 385.

Reply:
Done